.github/workflows/deploy.yml

```diff
@@ -34,7 +35,7 @@ jobs:
         uses: Swatinem/rust-cache@v2
 
       - name: Install Stellar
-        run: cargo install --locked stellar-cli --features opt
+        run: cargo install --locked stellar-cli@22.0.0-rc.1 --features opt
```

Comment:
Could we get this from Cargo.toml? Then we wouldn't need to remember to change this in future updates.